packages/cli/src/commands/hydrogen/build.ts

```diff
@@ -91,16 +92,24 @@ export async function runBuild({
 
     if (sizeMB >= 1) {
       output.warn(
-        '🚨 Worker bundle exceeds 1 MB! This can delay your worker response.\n',
+        `🚨 Warning: worker bundle exceeds 1 MB! This can delay your worker response.${
+          // @ts-ignore
+          remixConfig.serverMinify
+            ? ''
+            : ' Minify your bundle by adding `serverMinify: true` to remix.config.js.'
+        }\n`,
       );
     }
   }
 
   if (!disableRouteWarning) {
     const missingRoutes = findMissingRoutes(remixConfig);
     if (missingRoutes.length) {
+      const packageManager = await getPackageManager(root);
+      const exec = packageManager === 'npm' ? 'npx' : packageManager;
+
       output.warn(
-        '🚨 Standard Shopify routes missing; run `shopify hydrogen check routes` for more details.',
+        `🚨 Warning: standard Shopify routes missing; run \`${exec} shopify hydrogen check routes\` for more details.\n`,
```

Comment:
Just a heads up, I think eventually we can use `${outputToken.genericShellCommand('shopify hydrogen check routes')}`.

Reply:
I gave this a shot locally with an older version of that API and it made no different on the visual output, so I think we can just return to this later when the new APIs are available to us.